futures-core/src/task/__internal/atomic_waker.rs

```diff
@@ -1,8 +1,9 @@
 use core::fmt;
 use core::cell::UnsafeCell;
 use core::sync::atomic::AtomicUsize;
-use core::sync::atomic::Ordering::{Acquire, Release, AcqRel};
+use core::sync::atomic::Ordering::{Acquire, Release, AcqRel, SeqCst};
 use crate::task::{LocalWaker, Waker};
+use std::sync::atomic::Ordering::Relaxed;
```

Comment:
join this w/ the imports above (under `core::sync`) to make travis happy

Reply:
Fixed, thanks!